tests/integration/test_basic.py

```diff
@@ -214,3 +214,23 @@ def test_compression(mini_sentry, relay, content_encoding):
         data=encodings[content_encoding](b'{"message": "hello world"}'),
     )
     response.raise_for_status()
+
+
+@pytest.mark.parametrize(
+    "cross_origin_resource_policy",
+    [
+        "cross-origin",
+    ],
+)
+def test_corp_response_header(mini_sentry, relay, cross_origin_resource_policy):
+    project_id = 42
+    mini_sentry.add_basic_project_config(project_id)
+    relay = relay(mini_sentry)
+
+    response = relay.post(
+        "/api/42/store/?sentry_key=%s" % mini_sentry.get_dsn_public_key(project_id),
```

Comment:
I think you could write it something like this

```suggestion
        f"/api/42/store/?sentry_key={mini_sentry.get_dsn_public_key(project_id)}",
```

Reply:
I like this as well. I was following what looked like convention from the rest of the tests. Happy to accept this suggestion. :)